pkg/kubeconfig/kubecfg_builder.go

```diff
@@ -45,10 +45,14 @@ type KubeconfigBuilder struct {
 }
 
 // Create new KubeconfigBuilder
-func NewKubeconfigBuilder() *KubeconfigBuilder {
-	c := &KubeconfigBuilder{}
-	c.configAccess = clientcmd.NewDefaultPathOptions()
-	return c
+func NewDefaultKubeconfigBuilder() *KubeconfigBuilder {
+	return NewKubeconfigBuilder(clientcmd.NewDefaultPathOptions())
+}
+
+func NewKubeconfigBuilder(configAccess clientcmd.ConfigAccess) *KubeconfigBuilder {
```

Comment:
I think it's OK to just add the argument to NewKubeconfigBuilder, we're breaking callers anyway with the rename.  Even if we weren't renaming the function, I personally like to break the signature so that I have to think about each callsite.

Reply:
sorry I totally forgot about my PR and your comment, let me fix this.